tree/master/cloud/src/solution/Deployment/Scripts/DeployOfficeApp.ps1

```diff
@@ -191,22 +192,40 @@ $ScriptDirectory = (ScriptRoot)
 Function Get-ParentDirectory {Split-Path -Parent(Split-Path $MyInvocation.ScriptName)}
 $ParentDirectory = (Get-ParentDirectory)
 
+# Set  folder paths
+$RootPath = Split-Path(Split-Path (Split-Path $PSScriptRoot -Parent) -Parent) -Parent
+$DeployPath = "$RootPath\deployments"
+$HelperPath = "$RootPath\deployments\scripts\Helper Utilities"
+$ExternalPath = "$RootPath\Helper Utilities\External"
+$DeployPath = "$RootPath\solution"
+
 #Set Excel file path, uncomment below line if you want to use this script separately
 #$ExcelFilePath = "$ParentDirectory\MCDeploymentConfig.xlsx"
+#$ExcelFilePath = "C:\Repos\mattercenter2\tree\master\cloud\src\deployments\deployments\MCDeploymentConfig.xlsx"
```

Comment:
@AISDUG I think no paths should be hard coded and it should be read from config file